docs_user/modules/openstack-barbican_adoption.adoc

```diff
@@ -0,0 +1,100 @@
+= Barbican adoption
+
+Adopting Barbican means that an existing `OpenStackControlPlane` CR, where Barbican
+is initialy disabled, should be patched to start the service with the configuration
+parameters provided by the source environment.
+
+When the procedure is over, the expectation is to see the `BarbicanAPI`, `BarbicanWorker`,
+`BarbicanKeystoneListener` services are up and running.
+`Keystone endpoints` should also be updated and the same crypto plugin of the source Cloud
+will be available. If the conditions above are met, the adoption is considered concluded.
+
+____
+*NOTE* This procedure configures barbican to use the simple_crypto backend.
+Additional backends are available, such as PKCS11 and DogTag, however they are
+not supported in this release.
+____
+
+== Prerequisites
+
+* Previous Adoption steps completed. Notably, MariaDB and Keystone and xref:ovn_adoption.adoc[OVN DBs]
```

Comment:
xref:ovn_adoption.adoc[OVN DBs]  not sure what this is supposed to be, but we don't depend on OVN , do we?